# HG changeset patch
# User Mike Gilbert <floppym@gentoo.org>
# Date 1399130996 14400
#      Sat May 03 11:29:56 2014 -0400
# Node ID f8f63e119bd85b16bdee9b95195aba60d5fbca27
# Parent  60c8bb0cd34632becb758a063032e76c13dc3425
Remove unicode literal to avoid syntax error in python3.2

diff --git a/cherrypy/test/test_config.py b/cherrypy/test/test_config.py
--- a/cherrypy/test/test_config.py
+++ b/cherrypy/test/test_config.py
@@ -206,7 +206,7 @@
 
         if not compat.py3k:
             self.getPage("/repr?key=thing3")
-            self.assertBody(repr(u'test'))
+            self.assertBody(repr(unicode('test')))
 
         self.getPage("/repr?key=complex")
         self.assertBody("(3+2j)")
